compiler/src/codegen/compcore.re

```diff
@@ -378,7 +378,7 @@ let grain_number_min = (-0x3fffffff); // 0xC0000001
 
 /** Constant compilation */
 
-let rec compile_const = c: Literal.t => {
+let rec compile_const = (c): Literal.t => {
```

Comment:
Is there a prettifier somewhere that we could use to catch things like this and make it part of our CI? Now that I say that out loud, I feel like I am forgetting a thread somewhere about this...

Reply:
Yeah, we have a `esy format` but I haven't proxied that to the yarn commands yet or set up CI because I think the Reason formatter will update a bunch and I was waiting until some of the bigger PRs landed to avoid conflicts.

It might be time though because the exceptions PR still needs to be converted to Reason AFAIK